src/app/components/crypto-assets/bitcoin/runes-asset-list/runes-asset-item.layout.tsx

```diff
@@ -11,8 +12,8 @@ interface RunesAssetItemLayoutProps {
   rune: RuneToken;
 }
 export function RunesAssetItemLayout({ rune }: RunesAssetItemLayoutProps) {
-  const balance = rune.balance.amount.toString();
-  const formattedBalance = formatBalance(balance);
+  const balanceAsString = convertAmountToBaseUnit(rune.balance).toString();
+  const formattedBalance = formatBalance(balanceAsString);
```

Comment:
This is problematic, why doesn't `formatBalance` accept `Money`?

Reply:
Good point, I can change this.

Reply:
Really not sure I understand needing to change this when we need the string to display at this point? Why keep it as `Money` just to display it?